Packs/AtlassianConfluenceCloud/Integrations/AtlassianConfluenceCloud/AtlassianConfluenceCloud.py

```diff
@@ -1364,6 +1377,40 @@ def get_events(client: Client, args: dict) -> tuple[list[dict], CommandResults]:
         outputs_prefix=OUTPUT_PREFIX["EVENT"],
         readable_output=tableToMarkdown("Events", t=events, removeNull=True),
     )
+    
+def confluence_cloud_content_get_command(client: Client, args: dict[str, str]) -> CommandResults:
+    """
+    Execute the confluence-cloud-content-get command. Fetches content details from Confluence Cloud by content ID.
+
+    :type client: ``Client``
+    :param client: The client used to send HTTP requests to the Confluence Cloud API.
+
+    :type args: ``Dict[str, str]``
+    :param args: The command arguments provided by the user.
+
+    :return: ``CommandResults`` containing the content details.
+    :rtype: ``CommandResults``
+    """
+    validate_get_content_args(args)
+
+    content_id = args["content_id"]
+    params = {"body-format":"raw"}
+    request_url = URL_SUFFIX["CONTENT"] + f"/{content_id}"
+    params = {"expand":"body.storage"}
+
+    response = client.http_request(method="GET", url_suffix=request_url, params=params)
+    response_json = response.json()
```

Comment:
if the search returned not found, the command should not return an error, the readable should states that the content was not found and not fail the command, please verify this case